OrbitGl/CaptureWindow.cpp

```diff
@@ -376,13 +376,14 @@ void CaptureWindow::MouseWheelMoved(int a_X, int a_Y, int a_Delta,
   if (a_Delta == 0) return;
 
   // Normalize and invert sign, so that delta < 0 is zoom in.
-  int delta = a_Delta < 0 ? 1 : -1;
+  const int delta = a_Delta < 0 ? 1 : -1;
+  const auto delta_float = static_cast<float>(delta);
 
   if (delta < m_MinWheelDelta) m_MinWheelDelta = delta;
   if (delta > m_MaxWheelDelta) m_MaxWheelDelta = delta;
 
-  float mousex = a_X;
-  float mousey = a_Y;
+  const auto mousex = static_cast<float>(a_X);
+  const auto mousey = static_cast<float>(a_Y);
```

Comment:
Nit: This looks surprising to me. Making the copy of `a_X` and `a_Y` here makes me think about why. And that makes me believe that ScreenToWorld changes the value of `a_X` and `a_Y`. What about either removing those variables or at least minimizing their scope by moving down?

Reply:
> > > I see a lot of things I would change here, but that's all unrelated to your change. :-) So LGTM!
> > > I added to nitty-gritty comments though.
> > 
> > 
> > Right. As I am not that familiar with the overall code here, I just wanted to fix the warning for now and do some obvious improvements without changing the code's structure (i.e. adding the const's).
> > Though I suggest for now, I will address your comments and we should be fine.
> > Please also note, that this PR is targeting the release branch.
> 
> Ah I didn't notice that. Why do you want to target the release branch with a compilation fix?

As we said, that we wanted to keep the release branches for later, it feels super unnatural to have one, that might not compile. Assuming in future, someone checks out that branch, and compilation fails. This would be unfortunate. Thus, I suggest adding this fix to the release. 

Btw, I addressed your comments.